rts/Lua/LuaHandle.cpp

```diff
@@ -475,12 +480,13 @@ bool CLuaHandle::RunCallInTraceback(lua_State* L, const LuaHashString& hs, int i
  *
  * @function LoadCode
  */
-bool CLuaHandle::LoadCode(lua_State* L, const string& code, const string& debug)
+bool CLuaHandle::LoadCode(lua_State* L, std::string code, const string& debug)
```

Comment:
Wouldn't have `std::sting& code` have been preferable here?

Reply:
Just taking the value, not reference, is expressing the "this functions consumes code and does something with it" better then just reference, it's not clear from function declaration how the referenced string would be modified.